packages/@lwc/engine-core/src/framework/main.ts

```diff
@@ -37,6 +37,7 @@ export { unwrap } from './membrane';
 export { sanitizeAttribute } from './secure-template';
 export { getComponentDef, isComponentConstructor } from './def';
 export { profilerControl as __unstable__ProfilerControl } from './profiler';
+export { getUpgradableConstructor } from './upgradable-element';
```

Comment:
this needs to be exposed so engine-dom can rely on it, not very happy about this, but it is what it is... we will clean this a lot more once we have the node-reactions in place

Reply:
the good news is that this give you no powers.